code/os/02-memanagement/os.ld

```diff
@@ -104,8 +104,15 @@ SECTIONS
 		 * current location becomes aligned on 4096-byte boundary.
 		 * This is because our paging system's resolution is 4,096 bytes.
 		 */
-		. = ALIGN(4096);
-		PROVIDE(_data_start = .);
+		/* . = ALIGN(4096);
+		PROVIDE(_data_start = .); */
+		/**! BUG Here is a question, if `. = ALIGN (4096)` in the front, 
```

Comment:
Rootcasue analysis should be in commit message, not in code changes.

Reply:
Yes, I explained that in pr comment, the comment analysis in the code was to record the situation at that time